src/include/sof/audio/audio_stream.h

```diff
@@ -36,8 +36,8 @@ struct audio_stream {
 
 	/* runtime stream params */
 	uint32_t frame_fmt;	/**< enum sof_ipc_frame */
```

Comment:
typo in commit subject. s/audio_strean/audio_stream

Reply:
Thanks, changed